README.md

```diff
@@ -63,7 +70,6 @@ dependencies {
 WIRE_SDK_USER_ID=abcd-1234-efgh-5678
 WIRE_SDK_EMAIL=your_email@domain.com
 WIRE_SDK_PASSWORD=randomPassword
-WIRE_SDK_CLIENT=dd228f6343d3916
 WIRE_SDK_ENVIRONMENT=my.domain.link
 ```
```

Comment:
On my PR : https://github.com/wireapp/wire-apps-jvm-sdk/pull/164 I moved this Environment variables to `APPLICATION.md` when explaining about how to run the SDK